README.md

```diff
@@ -245,6 +247,7 @@ Option 1 is typically preferred by those users who already have automated deploy
 The GraphiteMetricsConsumer can be registered and configured by adding a snippet similar to the following to
 `storm.yaml` (see below) and by configuring the destination Graphite server appropriately (not shown here).
 
+##### Reporting Metrics to Graphite
```

Comment:
We should mention here (Graphite section) that currently you can report metrics to either Graphite or Kafka, but not to both at the same time.